3_javascript/javascript1/week3/JS1-W3

```diff
@@ -0,0 +1,121 @@
+////////////////////////////Item array removal
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
+  }
+}
+
+console.log(names);
+
+////////////////////////////When will we be there??
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+  function timeCalculator() {
+    let travelTimeInHours = travelInformation.destinationDistance / travelInformation.speed;
+    const hours = Math.floor(travelTimeInHours);
+    const minutes = Math.round((travelTimeInHours - hours) * 60);
+    
+    return {
+      hours,
+      minutes,
+    };
+  }
+  
+  const { hours, minutes } = timeCalculator(travelInformation);
+  
+  console.log(`${hours} hours and ${minutes} minutes`);
+  
+////////////////////////////Series duration of my life
+
+const seriesDurations = [
+    {
+      title: "Game of thrones",
+      days: 3,
+      hours: 1,
+      minutes: 0,
+    },
+    {
+      title: "Handmaids Tale",
+      days: 1,
+      hours: 22,
+      minutes: 40,
+    },
+    {
+      title: "Westworld",
+      days: 3,
+      hours: 10,
+      minutes: 40,
+    },
+  ];
+
+  function logOutSeriesText() {
+    let totalMinutesWatched = 0;
+
+    for (const series of seriesDurations) {
+      const { days, hours, minutes } = series;
+      const seriesDurationInMinutes = days * 24 * 60 + hours * 60 + minutes;
+      totalMinutesWatched += seriesDurationInMinutes;
+
+      const yearsWatched = totalMinutesWatched / (60 * 24 * 365); 
+      const percentageOfLife = (yearsWatched / 80) * 100; 
+
+      console.log(
+        `${series.title} took ${percentageOfLife.toFixed(2)}% of my life.`
+      );
+    }
+
+    console.log(`In total, that is ${totalMinutesWatched} minutes of my life.`);
+  }
+logOutSeriesText();
+
```

Comment:
👍 Your strategy for computing the total time of each series in minutes is spot on. It's a great way to get an exact total.
⚠️ Just a heads up! Currently, the percentage calculation in your loop seems to cumulatively increase for each series, which might not give the correct percentage for individual series.
🧐 A quick tip for clarity: Some inline comments explaining certain values (like why 80 was used) can be super helpful for anyone reading the code.